XenModel/Messages.resx

```diff
@@ -2527,6 +2527,9 @@ Do you want to assign it to the schedule '{2}' instead?</value>
   <data name="CHECKING_CANEVACUATE_STATUS" xml:space="preserve">
     <value>Checking VM migration status</value>
   </data>
+  <data name="CHECKING_CHECK_FOR_UPDATE_STATUS" xml:space="preserve">

```

Comment:
We don't say "check a check", but rather "run a check" or "check something for blah blah", so you would rename this to `CHECKING_UPDATE_FILE_STATUS` and the message to `Checking update file status`. However, I don't think we need to expose this level of detail, but rather say `Searching for available updates`. Similarly CHECK_FOR_UPDATE_AVAILABILITY_CHECK_DESCRIPTION=`Updates availability check`

Reply:
Please note that we are not exactly checking for "updates availability", because, as this is used in the RPU, we don't know the version the host will be upgraded to, so can't check if any updates are available on that version. This check purely checks if the CFU file can be downloaded, so if there is internet connection and the CFU website is reachable.